v63003/u_inref/gtm8735.csh

```diff
@@ -0,0 +1,48 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+#
+
+echo '# Creating database, manually setting variables so database is compatible with read only'
+setenv gtm_test_db_format "NO_CHANGE"
+setenv acc_meth MM
+setenv test_encryption NON_ENCRYPT
+source $gtm_tst/com/mm_nobefore.csh
+$gtm_tst/com/dbcreate.csh mumps 1 >>& create1.out
+$MUPIP SET -region DEFAULT -ACCESS_METHOD=MM -NOSTATS >>& settings.out
```

Comment:
After every MUPIP SET command, I would like the following to be done to verify the db file header does say Read only status is what the MUPIP SET command said.

dse dump -file |& $grep "Read Only"

Reply:
Since DSE opens and closes database files, might there be less overhead in running ``mupip dumpfhead -file <filename> | grep sgmnt_data.read_only`` or ``mupip dumpfhead -file <filename> | mumps -run %XCMD 'write:$zfind(%l,"sgmnt_data.read_only") %l,!``?

Reply:
This test is a very simplistic one and we are going to do around 4 DSE invocations. It should be an insignificant change. Moreover, the user is more likely to run DSE to check the "Read Only" flag setting. And so it is better for us to use the same tool (DSE DUMP) instead of MUPIP DUMPFHEAD.